k8s-manifests/ecommerce-app/frontend.yaml

```diff
@@ -69,10 +69,23 @@ metadata:
   name: frontend
 spec:
   ports:
-  - port: 3000
```

Comment:
This change won't work due to the ingress controller pointing to port 3000 down below.